drivers/platform/generic/delay.c

```diff
@@ -0,0 +1,72 @@
+/***************************************************************************//**
```

Comment:
this all looks like entirely new code;
well, from a patch perspective point of view;
why not move these file from an existing place?

the diff would be much smaller and easier to track;

Reply:
Well, not all the drivers were updated to use this new approach - moving the code will mean that the old platform_drivers gets removed, right?

Reply:
Isn't this entirely new code ? Did we have this format before, or everything was in platform_data ?

Reply:
I was mainly a copy-paste process.
Have a look here: https://github.com/analogdevicesinc/no-OS/blob/master/drivers/platform/generic/platform_drivers.c

Reply:
why not do this a sequential split?
or, remove the file when these files get introduced in the same commit

i don't insist on doing it;
but it does show up neater if the commit has the original/old file on which to anchor the "new" code/files

Reply:
Sorry, it was a misunderstanding. I've pushed some changes.